File: tests/specific-tests/async=nowait/0002.f
       program test
       integer i
!$acc parallel async(1)
       i = 0
!$acc end parallel
!$acc parallel async
       i = 0
!$acc end parallel
!$acc parallel async(2) wait
       i = 0
!$acc end parallel
!$acc wait(2)
       end program

File: tests/specific-tests/async=nowait/0002.c
int main (int argc, char *argv[])
{
	int x;

	#pragma acc parallel async(1)
	{
	x = 1;
	}

	_Pragma("acc parallel async(2)")
	{
	x = 2;
	}

	_Pragma("acc \
	 parallel async(3)")
	{
	x = 3;
	}

	#pragma acc wait (3)
	#pragma acc wait
}
